src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import Counter from "./components/Counter/Counter";
+import SearchForm from "./components/SearchForm/SearchForm";
+import Genres from "./components/Genres/Genres";
+import "./App.scss";
+
+const App: React.FC = () => {
+  // @TODO Add layout components and move SearchForm and Genres in their components
+
+  const handleSearch = (searchQuery: string) => {
+    // @TODO search logic here
+  };
+
+  const genres = ["All", "Documentary", "Comedy", "Horror", "Crime"];
```

Comment:
Since the list is a constant, you could move it outside of a React component so it is not redefined with every re-render. You could event create `constants.ts` file and import the list from there.